KomaMRIBase/src/sequences/ADC/design.jl

```diff
@@ -0,0 +1,26 @@
+"""
+"""
+function make_adc(num_samles; Δtadc=0, duration=0, delay=0,
+    freq_offset=0, phase_offset=0, dead_time=0, sys=nothing)
+
+    if !isnothing(sys)
+        dead_time = sys.ADC_dead_time_T
+        Δtadc = sys.ADC_Δt
+    end
```

Comment:
This should not be required, set the default for optional parameter sys = Scanner()